copypasta/README.md

```diff
@@ -0,0 +1,34 @@
+# copypasta
+
+copypasta is a (rust-clipboard)[https://github.com/aweinstock314/rust-clipboard] fork, adding support for the Wayland clipboard.
+
+rust-clipboard is a cross-platform library for getting and setting the contents of the OS-level clipboard.  
+
+## Example
+
+```rust
+extern crate copypasta;
+
+use copypasta::ClipboardContext;
+
+fn example() {
+    let mut ctx = ClipboardContext::new().unwrap();
+    println!("{:?}", ctx.get_contents());
+    ctx.set_contents("some string".to_owned()).unwrap();
```

Comment:
Maybe show a read here too.

Reply:
I'd like to keep the copypasta crate mostly unchanged for now. It has been months now without any response, but I still haven't completely given up on upstreaming our changes.

So at least for this release, I think it's better to mostly keep it untouched. If we decide to go over it again and modify/clean some stuff up, there's probably a few things that could be improved.

Reply:
Oh that's fair.